test/support/utils.rb

```diff
@@ -75,5 +75,9 @@ def create_pseudo_terminal
       p "terminal finished without reading 'y'"
       raise
     end
+
+    def strip_line_num(str)
+      str.gsub(/ *\d+\| ?/, '')
```

Comment:
Perhaps an anchor in the regular expression would help?

```suggestion
      str.gsub(/^\s*\d+\| ?/, '')
```

Reply:
Yes, you are right. Thanks!